particula/aerosol_dynamics/parcel.py

```diff
@@ -48,43 +38,68 @@ def name(self) -> str:
     def add_particle(self, particle_object):
         """Adds a particle to the parcel.
         Parameters:
-            particle_data (list): The radius of the particle.
+            particle_object (object)
         """
-        self._particle_data.append(particle_object)
+        self._particle_data = np.append(self._particle_data, particle_object)
 
     def create_and_add_particle(
-        self, name: str, radius, density=1e3, charge=0
+        self, name: str, radius, density=1e3 * u.kg / u.m**3,
+        charge=0 * u.dimensionless
     ):
         """creats and then Adds a particle to the parcel.
         Parameters:
-            particle_data (list): The radius of the particle.
+            name    (str)   [no units],
+            radius  (float) [m]
+        Optional:
+            density (float) [kg/m**3]       default = 1e3 [kg/m**3],
+            charge  (int)   [dimensionless] default = 0 [dimensionless]
         """
-        self._particle_data.append(
+
+        try:  # maybe move to particle class
+            radius.check('m')
+        except AttributeError:
+            print('Please add units to radius. E.g., radius = 150 * u.nm]')
```

Comment:
Hm... We need to discuss this in more depth. Having the user supply units is risky! We should ideally instruct the reader to give us the radius in a specific unit, but we can discuss later and have a unified setup across all modules.

Reply:
I agree, this didn't feel like a good way to go about it as it was one-off and not getting to the root of the possible problems/error of having the wrong units.